bin/tbg

```diff
@@ -186,16 +191,17 @@ function run_cfg_and_set_solved_variables
     set_env_vars tbg_vars_overwritten
     all_tbg_vars=$(echo -e "$tbg_vars_overwritten\n$cfg_script_overwritten_vars")
     resolve_vars all_tbg_vars
+
     # evaluate all variable assignments which are contained in the tpl file
     # variables are already set in the environment and resolved
     while read -r data_set
     do
-        var_name=$(echo "$data_set" | cut -d"=" -f1)
+        var_name=$(echo -e "$data_set" | cut -d"=" -f1)
         var_value=$(eval echo \$$var_name)
         if [ -n  "$var_value" ] ; then
-            eval "$data_set"
+            export "$var_name=$var_value"
         fi
-    done < <(echo "$tbg_vars_overwritten" |  grep "^[[:blank:]]*[[:alpha:]][[:alnum:]_]*=.*")
+    done < <(echo "$all_tbg_vars" |  grep "^[[:blank:]]*[[:alpha:]][[:alnum:]_]*=.*")
```

Comment:
@psychocoderHPC why this change from `$tbg_vars_overwritten` to `$all_tbg_vars`?

Reply:
With the new changes it is required to evaluate and register all variables again and not only the overwritten.